### DISCUSSION TITLE
How can i switch to different datadir based on select input

### DISCUSSION THREAD

**Discussion by Developermayank** [pocketbase/pocketbase#5984] [Q&A] [answered]
I am creating an electron app with svelte. I am facing an issue with serving the pocketbase instance with dynamically data path.

I want to switch to different data directory on selection an option on my svelte select input.

**Comment by ganigeorgiev** [comment, marked as answer]
The `pb_data` directory cannot change while the application is running.

There are not enough details to understand your use case and what you are trying to do, but if you really need it then you can try calling [`app.Bootstrap()`](https://pkg.go.dev/github.com/pocketbase/pocketbase@v0.23.3/core#BaseApp.Bootstrap) or [`app.Restart()`](https://pkg.go.dev/github.com/pocketbase/pocketbase@v0.23.3/core#BaseApp.Restart) (the latter uses `execve` to restart and replace the current running PocketBase process).
